cmd/valid/README.md

```diff
@@ -0,0 +1,69 @@
+# valid
```

Comment:
How about moving the program to `utf8/cmd/valid`? Or naming it something with `utf8` in it?